<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistogramEva</class>
 <widget class="QDialog" name="HistogramEva">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1275</width>
    <height>590</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1275</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1275</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCustomPlot" name="histogram_AE" native="true">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>30</y>
     <width>1260</width>
     <height>251</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1260</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1260</width>
     <height>16777215</height>
    </size>
   </property>
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>0</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/new/prefix1/img/PD.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QCustomPlot" name="histogram_EB" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>1260</width>
     <height>251</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1260</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1260</width>
     <height>16777215</height>
    </size>
   </property>
   <widget class="QLabel" name="imageLabel_2">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>10</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>0</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/new/prefix1/img/PD.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Алиса -&gt; Ева</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>91</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Ева -&gt; Боб</string>
   </property>
  </widget>
  <widget class="QScrollBar" name="horizontalScrollBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>570</y>
     <width>1260</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>../../qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
